Sources/Rendering/Misc/SynchronizableRenderWindow/index.js

```diff
@@ -225,9 +225,11 @@ export function vtkSynchronizableRenderWindow(publicAPI, model) {
 
   const addOn = createSyncFunction(publicAPI, model.synchronizerContext);
 
-  Object.keys(addOn).forEach((methodName) => {
-    publicAPI[methodName] = addOn[methodName];
-  });
+  Object.keys(addOn)
+    .filter(methodName => methodName.indexOf('SynchronizedViewId') < 0)
```

Comment:
I would expand the `set` method in [macro](https://github.com/Kitware/vtk-js/blob/master/Sources/macro.js#L141) like below:

```
publicAPI.set = (map = {}, noWarning = false, noFunction = false) => {
    let ret = false;
    Object.keys(map).forEach((name) => {
      const fn = noFunction ? null : publicAPI[`set${capitalize(name)}`];
      if (fn && Array.isArray(map[name])) {
        ret = fn(...map[name]) || ret;
      } else if (fn) {
        ret = fn(map[name]) || ret;
      } else {
        // Set data on model directly
        if (['mtime'].indexOf(name) === -1 && !noWarning) {
          vtkWarningMacro(`Warning: Set value to model directly ${name}, ${map[name]}`);
        }
        model[name] = map[name];
        ret = true;
      }
    });
    return ret;
  };
```